roles/nova-common/templates/etc/nova/nova.conf

```diff
@@ -150,7 +150,12 @@ preallocate_images = {{ nova.preallocate_images }}
 connection=mysql://nova:{{ secrets.db_password }}@{{ endpoints.db }}/nova?charset=utf8
 
 [libvirt]
+{% if ansible_architecture != "ppc64le" -%}
 virt_type={{ nova.libvirt_type }}
+{% elif ansible_architecture == "ppc64le" -%}
+virt_type=kvm
+{% endif -%}
+
```

Comment:
I don't understand this. You should be able to define what the `libvirt_type` is in a host-level config file. Every ppc64le host could hard code it to kvm there, and then this template can still use `nova.libvirt_type`.